client/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "musiqparti-client",
   "version": "0.1.0",
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 9001",
```

Comment:
just curious, why port 9001?

Reply:
9001 is just a random one I picked. No specific reason, I can change. The main thing I was trying to do is just factor this stuff out to the configs.

And which change were you referring to? In the server's port is defined with env variables like this, right?
```
app.listen(process.env.PORT)
```
I think this makes sense, and we should probably keep that.

I just couldn't figure out how to set up the port over env variables in the client